project/Build.scala

```diff
@@ -19,7 +19,7 @@ object Build extends sbt.Build {
 
   lazy val basicSettings = Seq(
     organization := "com.wandoulabs.avro",
-    version := "0.1.3-talend-SNAPSHOT",
+    version := "0.1.4-talend-SNAPSHOT",
```

Comment:
TODO: Make a release.

Reply:
Just for info, current version being sent with processing-beam is `0.1.3-talend`, which is just a re-labeled `0.1.3-talend-SNAPSHOT`